example-get-started/generate.sh

```diff
@@ -127,27 +128,35 @@ git commit -m "Create ML pipeline stages"
 git tag -a "7-ml-pipeline" -m "ML pipeline created."
 dvc push
 
+
 dvc run -n evaluate \
   -d src/evaluate.py -d model.pkl -d data/features \
-  -M scores.json \
-  --plots-no-cache prc.json \
-  --plots-no-cache roc.json \
-  python src/evaluate.py model.pkl data/features scores.json prc.json roc.json
-dvc plots modify prc.json -x recall -y precision
-dvc plots modify roc.json -x fpr -y tpr
-git add .gitignore dvc.yaml dvc.lock prc.json roc.json scores.json
+  -M evaluation.json \
+  --plots-no-cache evaluation/plots/precision_recall.json \
+  --plots-no-cache evaluation/plots/roc.json \
+  --plots-no-cache evaluation/plots/confusion_matrix.json \
+  --plots evaluation/importance.png \
```

Comment:
Not sure what's in scope, but should we add titles?

Reply:
specifically for this image? I had it, then decided that Y axis label is kinda enough to understand, wdyt?

Reply:
Yeah, I meant for the plots generally, but you are right. It took me a second to separate ROC/PRC, but a title will just duplicate the axis names.